cxx/src/lib/utility/ProcessingHistory.cc

```diff
@@ -750,6 +750,7 @@ ProcessingHistory& ProcessingHistory::operator=(const ProcessingHistory& parent)
     mytype=parent.mytype;
     algorithm=parent.algorithm;
     algid=parent.algid;
+    elog=parent.elog;
```

Comment:
This should be a bug. I guess our tests are outdated so it is never detected. I need to revise corresponding test.

Reply:
This operator is never explicitly called on the python side. I added a test for the `elog` member instead.